Assets/Runtime/ScreenTransitions/SimpleFadeTransition.cs

```diff
@@ -20,27 +20,33 @@ public class SimpleFadeTransition : ATransitionComponent
         private Action currentAction;
         private Transform currentTarget;
 
-        private float startValue;
-        private float endValue;
+        private float startValue = 0f;
```

Comment:
what is the point set some value? it anyhow will be overwritten on `Animate()`

Reply:
no point, just tested some things and didnt removed